exporter/stackdriverexporter/observability.go

```diff
@@ -25,15 +25,8 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-// TODO: re-think if processor should register it's own telemetry views or if some other
-// mechanism should be used by the collector to discover views from all components
-
-func init() {
-	view.Register(viewPointCount)
-}
-
 var (
-	pointCount = stats.Int64("otelcol/stackdriver/point_count", "Count of metric points written to Cloud Monitoring.", "1")
+	pointCount = stats.Int64("otelcol/cloudmonitoring/point_count", "Count of metric points written to Cloud Monitoring.", "1")
```

Comment:
LGTM, but I wonder if we should call it `googlecloudmonitoring`, otherwise it might be confusing which "cloud" does this refer to. We should think about the same for renaming the exporter, e.g. `googlecloudmonitoringexporter` and `googlecloudtracexporter`.